helm/chart/crds/lifecycle.keptn.sh_keptnapps.yaml

```diff
@@ -0,0 +1,148 @@
+---
```

Comment:
should these files be generated by the build based on the different components?

Reply:
this is generated by helm package command in the makefile